examples/with-tailwindcss/package.json

```diff
@@ -12,9 +12,11 @@
     "react-dom": "^16.13.1"
   },
   "devDependencies": {
-    "postcss-flexbugs-fixes": "4.2.1",
+    "autoprefixer": "10.0.2",
+    "postcss": "8.1.7",
+    "postcss-flexbugs-fixes": "5.0.1",
     "postcss-preset-env": "^6.7.0",
```

Comment:
Might want to remove postcss-preset-env as it's not compatible with PostCSS 8 https://github.com/csstools/postcss-preset-env/issues/191

Reply:
Yeah, you are right! Missed that, fixed now. Thanks!

Reply:
I removed the `postcss-flexbugs-fixes` plugin as well, IE support has been dropped in v2 (https://tailwindcss.com/docs/upgrading-to-v2#support-for-ie-11-has-been-dropped)